src/main/java/org/spongepowered/common/mixin/core/entity/EntityMixin.java

```diff
@@ -558,21 +544,22 @@ private void onSpongeReadFromNBT(final CompoundNBT compound, final CallbackInfo
 
     }
 
-    @Redirect(method = "doBlockCollisions", at = @At(value = "INVOKE", target = "Lnet/minecraft/block/Block;onEntityCollision(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/block/state/IBlockState;Lnet/minecraft/entity/Entity;)V")) // doBlockCollisions
-    private void spongeImpl$onEntityCollideWithBlockState(
-        final Block block, final net.minecraft.world.World world, final BlockPos pos, final BlockState state, final Entity entity) {
+    @Redirect(method = "doBlockCollisions", at = @At(value = "INVOKE",
+        target = "Lnet/minecraft/block/BlockState;onEntityCollision(Lnet/minecraft/world/World;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/entity/Entity;)V")) // doBlockCollisions
```

Comment:
If you're moving this to a new line, change the `at = @At` to new line, so like:
```java
@Redirect(method = "doBlockCollisions,
    at = @At(value = "INVOKE",
        target = "Lnet/n...."
    )
)
```
This way it's clearer, at least it's the format I'm trying to migrate the codebase to.